modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/src/main/resources/META-INF/resources/js/components/list/List.es.js

```diff
@@ -22,6 +22,14 @@ import {SidebarContext} from '../sidebar/SidebarContext.es';
 export default ({data, field, totalEntries, type}) => {
 	const {toggleSidebar} = useContext(SidebarContext);
 
+	const formatDate = (field) => {
```

Comment:
@alinedoleron, why don't you added the commits [1](https://github.com/natocesarrego/liferay-portal/pull/629/commits/d0b5925a1ded31a8eaef31673acf5cccb752b65e) and [3](https://github.com/natocesarrego/liferay-portal/pull/629/commits/d7bc4fb8787593605309e22fd11ca7b1e92594ce) on this current commit? After all, they're very connected.